### PR TITLE
travis: drop OSX environment

### PR DESCRIPTION
**Pull request by daviddrysdale**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -60,6 +50,7 @@ install:
 
 before_script:
   - yes | ./scripts/resetdb.sh
+  - rvm get head # https://github.com/travis-ci/travis-ci/issues/6307
```
Comment (codingllama) [inline .travis.yml]: (Drive-by): Isn't this only useful for OSX builds?  (Sorry if I missed something, I skimmed the issue rather quickly.)

Comment (daviddrysdale) [reply .travis.yml]: oops, detritus from a previous attempt -- I'll remove
